Sources/FoundationEssentials/Data/Data+Reading.swift

```diff
@@ -25,6 +25,9 @@ import Glibc
 #elseif os(Windows)
 import CRT
 import WinSDK
+#elseif os(WASI)
+import WASILibc
+import var WASILibc.errno
```

Comment:
You mentioned

> we need to fix it in overlay side anyway

If this needs to be fixed on the overlay side anyways, should we just fix it there rather than disambiguating here?

Reply:
Ok, I'll fix the overlay side instead of introducing the immediate fix here :+1: